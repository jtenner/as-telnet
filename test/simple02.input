# IAC GA
%FF%F9

# IAC DO BINARY
%FF%FD%00