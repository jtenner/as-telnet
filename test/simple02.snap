IAC: %F9
SEND: %FF%FB%00
NEGOTIATE: DO %00
